src/entities/account.ts

```diff
@@ -1,9 +1,14 @@
-export function countCryptos(account: Account): number {
+function countCryptos(account: Account): number {
   return account.assets
     .reduce((sum, a) => a.value.t !== 'Store' ? sum + 1 : sum, 0);
 };
 
-export function countNFTs(account: Account): number {
+function countNFTs(account: Account): number {
   return account.assets
     .reduce((sum, a) => a.value.t === 'Store' ? sum + 1 : sum, 0);
 }
+
+export const accountModel = {
+  countCryptos,
+  countNFTs,
+};
```

Comment:
Not the best solution in terms of tree-shaking. It would be *a bit* more efficient to export function separately.

Reply:
This is one of the necessary sacrifices. Since the logic is divided into modules with strong coupling inside, this does not lead to big problems with the weight of imports.